src/components/LineChart.vue

```diff
@@ -1,7 +1,14 @@
 <template>
-  <Line
-    :options="chartOptions"
-    :data="chartData"/>
+  <div class="line-chart">
+    <Line
+      v-if="hasChart"
+      :options="chartOptions"
+      :data="chartData"/>
+    <blank-state
```

Comment:
added loader when changing values of chart and blank state indicator for all charts